Sources/Sentry/SentrySystemEventsBreadcrumbs.m

```diff
@@ -0,0 +1,174 @@
+#import "SentrySystemEventsBreadcrumbs.h"
+#import "SentrySDK.h"
+#import "SentryLog.h"
+
+// all those notifications are not available for tvOS
+#if TARGET_OS_IOS
+#    import <UIKit/UIKit.h>
+#endif
+
+@implementation SentrySystemEventsBreadcrumbs
```

Comment:
`h`: Can we please add tests for this before we merge this.

Reply:
tests have been added